README.md

```diff
@@ -50,15 +50,17 @@ artifact.platform_version # => "10.10"
 ```
 
 ## Unstable channel
-The `:unstable` channel is currently only available when connected to Chef's internal network.
-Configure Artifactory access by setting the following environment variables:
+The `:unstable` channel is currently only available when connected to Chef's internal network. mixlib-install uses a default set of read-only credentials when using :unstable channel. You can override these credentials by setting these environment variables:
+
 ```
 export ARTIFACTORY_USERNAME='username'
 export ARTIFACTORY_PASSWORD='password'
 ```
```

Comment:
Do we even need to offer this option? I think we should purge these env variables from the code base.